README.md

```diff
@@ -156,6 +156,10 @@ searcher = PGPE(
 # Instantiate a standard output logger
 _ = StdOutLogger(searcher)
 
+# Instantiate a pickling logger that will periodically save
+# the current solution
```

Comment:
What does "current solution" mean in a population-based context? Do you mean the current best?

Reply:
Thank you for the feedback! I pushed a commit that brings more detailed explanations for the PicklingLogger in these example codes.